assets/app/scripts/filters/resources.js

```diff
@@ -15,27 +15,35 @@ angular.module('openshiftConsole')
     };
   })
   .filter('annotationName', function() {
+    // This maps an annotation key to all known synonymous keys to insulate
+    // the referring code from key renames across API versions.
+    var annotationMap = {
+      "deploymentConfig":         ["openshift.io/deployment-config.name"],
+      "deployment":               ["openshift.io/deployment.name"],
+      "pod":                      ["openshift.io/deployer-pod.name"],
+      "deployerPodFor":           ["openshift.io/deployer-pod-for.name"],
+      "deploymentStatus":         ["openshift.io/deployment.phase"],
+      "deploymentStatusReason":   ["openshift.io/deployment.status-reason"],
+      "deploymentCancelled":      ["openshift.io/deployment.cancelled"],
+      "encodedDeploymentConfig":  ["openshift.io/encoded-deployment-config"],
+      "deploymentVersion":        ["openshift.io/deployment-config.latest-version"],
+      "displayName":              ["openshift.io/display-name"],
+      "description":              ["openshift.io/description"],
+      "buildNumber":              ["openshift.io/build.number"],
+      "buildPod":                 ["openshift.io/build.pod-name"]
+    };
     return function(annotationKey) {
-      // This maps an annotation key to all known synonymous keys to insulate
-      // the referring code from key renames across API versions.
-      var annotationMap = {
-        "deploymentConfig":         ["openshift.io/deployment-config.name"],
-        "deployment":               ["openshift.io/deployment.name"],
-        "pod":                      ["openshift.io/deployer-pod.name"],
-        "deployerPodFor":           ["openshift.io/deployer-pod-for.name"],
-        "deploymentStatus":         ["openshift.io/deployment.phase"],
-        "deploymentStatusReason":   ["openshift.io/deployment.status-reason"],
-        "deploymentCancelled":      ["openshift.io/deployment.cancelled"],
-        "encodedDeploymentConfig":  ["openshift.io/encoded-deployment-config"],
-        "deploymentVersion":        ["openshift.io/deployment-config.latest-version"],
-        "displayName":              ["openshift.io/display-name"],
-        "description":              ["openshift.io/description"],
-        "buildNumber":              ["openshift.io/build.number"],
-        "buildPod":                 ["openshift.io/build.pod-name"]
-      };
       return annotationMap[annotationKey] || null;
     };
   })
+  .filter('labelName', function() {
```

Comment:
@jwforres should I copy over the whole annotation map as a reasonable assumption for labels?